### ISSUE TITLE
Remove AWS Account ID from Prod bucket names

### ISSUE THREAD

**Issue by chuckwondo** [NASA-IMPACT/csdap-cumulus#50]
## Issue

Our buckets in Prod include the AWS Account ID in the names. While the configured bucket mapping hides this from appearing in the distribution URLs, the S3 URLs, which _do_ include the Account ID, are still exposed, so we want to use buckets without the Account ID in the names, to avoid exposing them (e.g., via the AWS S3 Access links shown in Earthdata Search).

## Acceptance Criteria

1. All of the Prod buckets used by Cumulus exclude the AWS Access ID from their names
2. A smoke test discovery/ingest workflow execution succeeds
3. The metadata published to the CMR for the smoke test granules no longer includes the AWS Account ID in the AWS S3 Access links (as verified in Earthdata Search)

**Comment by chuckwondo** [comment]
To apply fix to Production or to Dev deployments (primary UAT deployment is not affected, as its buckets do not include the account ID in the names), after #51 is approved and merged into main:

- Checkout and pull `main` branch
- Run `make docker` (just to be sure you're image is up to date)
- Run `make all-init`
- Run `make bash` (to open a terminal window within the Docker container)
- Within the container, run `aws s3 mv --recursive s3://csdap-${CUMULUS_STACK}-tfstate-$(aws sts get-caller-identity --query Account --output text)/${TS_ENV} s3://csdap-${CUMULUS_STACK}-tfstate/${TS_ENV}`
- Exit the container
- Run `make all-init` again (might not be necessary, but just in case, after moving the tfstate files above)
- Run `make all-up-yes`
